app/src/main/java/com/example/lexis/fragments/ArticleFragment.java

```diff
@@ -109,6 +97,7 @@ public void onClick(View textView) {
                     @Override
                     public void updateDrawState(TextPaint ds) {
                         super.updateDrawState(ds);
+                        // text should be black and not underlined
```

Comment:
This part is probably fine without additional commenting ^ Comments are useful in explaining more complex code logic or functions.